sonarqube-companion-rest/src/main/java/pl/consdata/ico/sqcompanion/sync/SyncAfterAppInitialization.java

```diff
@@ -20,6 +21,9 @@ public SyncAfterAppInitialization(final SynchronizationService synchronizationSe
 		this.synchronizationService = synchronizationService;
 	}
 
+	/**
+	 * TODO (mp): to nie powinno blokować wstawania aplikacji
```

Comment:
Ale w bazie masz tylko historię dla każdego projektu.
Ale nie masz listy projektów rozwiązanej względem zdalnego SQ.
Czy to dopisywałeś?

Te rzeczy są tylko w pamięci, bo póki co nie wiem czy, ani jak, to trzymać pomiędzy uruchomieniami ;-)

Co do samego zachowania to pewnie tak ale tylko pod warunkiem że będziemy mieli komplet danych, żeby nie było ryzyka że ktoś patrzy na puste ekrany. Wtedy bym wolał żeby był dłużej krecioł na starcie.
Pogadamy później, póki co merguj pan!